examples/dtls-echo/Makefile

```diff
@@ -8,68 +8,70 @@ BOARD ?= native
 RIOTBASE ?= $(CURDIR)/../..
 
 # TinyDTLS only has support for 32-bit architectures ATM
+# Note: for nrf52dk see issue #6022
 BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-uno chronos \
                    msb-430 msb-430h telosb waspmote-pro wsn430-v1_3b wsn430-v1_4 \
-                   z1 \
-                   nrf52dk # see #6022
+                   z1 nrf52dk
 
-BOARD_INSUFFICIENT_MEMORY := airfy-beacon calliope-mini cc2650stk maple-mini \
-                             microbit nrf51dongle nrf6310 nucleo32-f031 \
-                             nucleo32-f042 nucleo32-f303 nucleo32-l031 nucleo-f030 \
-                             nucleo-f070 nucleo-f072 nucleo-f103 nucleo-f302 nucleo-f334 \
-                             nucleo-l053 nucleo-l073 opencm904 pca10000 pca10005 \
-                             spark-core stm32f0discovery weio yunjia-nrf51822
+BOARD_INSUFFICIENT_MEMORY := airfy-beacon nrf51dongle nrf6310 nucleo-f103 \
```

Comment:
This was an omission from my part that was already identified at PR #6430. It's now addressed for this PR.

Also, the Issue #6022 is not anymore a factor for tinydtls. Therefore, I removed the nrf52dk board from the BOARD_BLACKLIST.

Yet, the new boards mips-malta, pic32-clicker and pic32-wifire are failing the buildtests, even with docker. Maybe because of the Issue #6588 ?